vsts/templates/pre-test-steps.yaml

```diff
@@ -3,8 +3,8 @@ parameters:
   repo_address: $(IOTHUB-E2E-REPO-ADDRESS)
   repo_user: $(IOTHUB-E2E-REPO-USER)
   repo_password: $(IOTHUB-E2E-REPO-PASSWORD)
-  image_edgeHub: mcr.microsoft.com/azureiotedge-hub:1.0.7
-  image_edgeAgent: mcr.microsoft.com/azureiotedge-agent:1.0.7
+  image_edgeHub: mcr.microsoft.com/azureiotedge-hub:1.0.6
+  image_edgeAgent: mcr.microsoft.com/azureiotedge-agent:1.0.6
```

Comment:
please remove all the changes under vsts/ from this PR.  It looks like your revert changed the files to some random version that isn't the version in master.

Reply:
updated from Master